app/assets/stylesheets/styles.scss

```diff
@@ -1,3 +1,36 @@
+$orange: #D45E00;
+$blue: #00bcd4;
+$red: #D4002E;
+
+
+.top-bar {
+  height: 50px;
```

Comment:
Properties should be ordered background-color, border-radius, box-shadow, font-family, height